dubbo-common/src/main/java/org/apache/dubbo/common/threadlocal/InternalThreadLocal.java

```diff
@@ -1,19 +1,19 @@
 /*
- * Copyright 2014 The Netty Project
```

Comment:
Why are there Netty headers here before? Please help confirm it.

Reply:
This is a special situation , because `InternalThreadLocal.java` is copied totally from `Netty`  project .  According to Apache's license rules, Netty's license header needs to be retained

Reply:
@lovepoem revert all netty license header